hack/ci/push-latest-cli/push-latest-cli.sh

```diff
@@ -34,36 +41,42 @@ SHORT_COMMIT="${COMMIT:0:8}"
 BUCKET="${BUCKET:-k8s-staging-kind}"
 # under each of these
 VERSIONS=(
-  latest
-  "${SHORT_COMMIT}"
+    "latest"
```

Comment:
these are arguments in an array, and we want the literal `latest` as a distinct word, quoting here is superfluous, as a literal it doesn't need double quotes